croutonjs/src/js/crouton-core.js

```diff
@@ -498,6 +498,14 @@ function Crouton(config) {
 					? meetingData[m]['location_info'].replace('/(http|https):\/\/([A-Za-z0-9\._\-\/\?=&;%,]+)/i', '<a style="text-decoration: underline;" href="$1://$2" target="_blank">$1://$2</a>')
 					: "";
 			meetingData[m]['map_word'] = self.localization.getWord('map').toUpperCase();
+			if (meetingData[m]['train_lines'] != null) {
```

Comment:
could you look for the presence of the glasses separator `#@-@#` and then handle that way versus binding to specific fields?

Reply:
that makes sense, then it would handle custom fields. like boat lines

Reply:
It should handle `ferry_lines` or `closest_blue_bikes_dock`